contrib/epee/include/net/net_helper.h

```diff
@@ -576,7 +577,14 @@ namespace net_utils
 				m_io_service.run_one();
 			}
 			// Ignore "short read" error
```

Comment:
I've not looked at the code but, is this at the end of session? If not, what good could come from ignoring a short read?